src/rzslider.js

```diff
@@ -2232,9 +2335,12 @@
         },
 
         onKeyboardEvent: function(event) {
-          var currentValue = this[this.tracking],
-            keyCode = event.keyCode || event.which,
-            keys = {
+          var keyCode = event.keyCode || event.which
+          var currentValue = this[this.tracking]
+          currentValue = this.options.skipRestrictedRangesWithArrowKeys
+            ? this.skipRestrictedRanges(keyCode, currentValue)
```

Comment:
This logic isn't working perfectly at the moment.
There are (at least) these issues:
- ![image](https://user-images.githubusercontent.com/2678610/169481833-35c7d986-84e6-4ff5-88ce-3e5e9bb1f93b.png) in this configuration, if I press the RIGHT key, I should go to 30. Here it goes to 31.
- ![image](https://user-images.githubusercontent.com/2678610/169481978-5976e4c7-f7e0-43da-9b87-6fd56866a4cc.png)  in this one, if I press the LEFT key, it should go to 19. Instead, it goes to 30.
- ![image](https://user-images.githubusercontent.com/2678610/169482121-5e3a79ee-8111-4c2d-8a8f-38466baac41e.png) here, if I press the RIGHT key, I should go to 31. Instead it goes to 20.

To avoid reimplementing a different logic that what's existing, I think you should add the new logic for multiple ranges inside the `getKeyActions()` instead.